src/assets/currencies/brl.svg

```diff
@@ -1,9 +1,32 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
-  <path fill="#6da544" d="M0 0h512v512H0z"/>
-  <path fill="#ffda44" d="M256 114.5L448 256 256 397.5 64 256z"/>
-  <circle cx="256" cy="256" r="80.8" fill="#f0f0f0"/>
-  <g fill="#0052b4">
-    <path d="M215.6 251a136 136 0 00-40.4 6 80.8 80.8 0 00147 45.4 136.2 136.2 0 00-106.6-51.5z"/>
-    <path d="M335.3 271.5A80.8 80.8 0 00181.7 224a166.9 166.9 0 01153.6 47.4z"/>
-  </g>
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.3.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#F0F0F0;}
+	.st1{fill:#6DA544;}
+	.st2{fill:#0052B4;}
+	.st3{fill:#FFDA44;}
+</style>
+<path class="st0" d="M335.3,271.5c-2.2,11.3-6.7,21.8-13.1,30.9c-25-31.3-63.5-51.4-106.6-51.4c-14,0-27.6,2.1-40.4,6.1
+	c0-0.3,0-0.7,0-1c0-11.3,2.3-22.1,6.6-31.9c10.9-2.2,22.2-3.5,33.8-3.5C262.5,220.6,305,240.1,335.3,271.5z"/>
+<path class="st1" d="M0,0v512h512V0H0z M256,397.5L64,256l192-141.5L448,256L256,397.5z"/>
+<g>
+	<path class="st2" d="M256,336.8c21,0,40.1-8,54.4-21.1C296.1,328.8,277,336.8,256,336.8c-44.3,0-80.3-35.6-80.8-79.8c0,0,0,0,0,0
+		C175.8,301.2,211.7,336.8,256,336.8z"/>
+	<path class="st2" d="M310.4,315.7c1.1-1,2.2-2,3.2-3.1C312.6,313.7,311.5,314.7,310.4,315.7z"/>
+	<path class="st2" d="M322.2,302.4c-0.4,0.6-0.8,1.2-1.3,1.8C321.3,303.5,321.8,303,322.2,302.4L322.2,302.4z"/>
+	<path class="st2" d="M335.3,271.5c0.1-0.4,0.2-0.8,0.2-1.2C335.5,270.7,335.4,271.1,335.3,271.5
+		C335.3,271.5,335.3,271.5,335.3,271.5z"/>
+	<path class="st2" d="M335.9,268.3c0.6-4,0.9-8.1,0.9-12.3c0-44.6-36.2-80.8-80.8-80.8c-33.3,0-61.9,20.1-74.2,48.9c0,0,0,0-0.1,0
+		c12.4-28.8,41-48.9,74.3-48.9c44.6,0,80.8,36.2,80.8,80.8C336.8,260.2,336.5,264.3,335.9,268.3z"/>
+</g>
+<path class="st3" d="M256,114.5L64,256l192,141.5L448,256L256,114.5z M256,336.8c-44.6,0-80.8-36.2-80.8-80.8s36.2-80.8,80.8-80.8
+	s80.8,36.2,80.8,80.8S300.6,336.8,256,336.8z"/>
+<g>
+	<path class="st2" d="M215.6,250.9c-14.1,0-27.6,2.1-40.4,6.1c0.6,44.2,36.5,79.8,80.8,79.8c27.4,0,51.6-13.6,66.2-34.5
+		C297.2,271.1,258.7,250.9,215.6,250.9z"/>
+	<path class="st2" d="M335.3,271.5c1-5,1.5-10.2,1.5-15.5c0-44.6-36.2-80.8-80.8-80.8c-33.3,0-61.9,20.2-74.3,48.9
+		c10.9-2.3,22.3-3.5,33.9-3.5C262.5,220.6,305,240.1,335.3,271.5z"/>
+</g>
```

Comment:
This is unnecessary big. A simple pass through SVGO (https://jakearchibald.github.io/svgomg/) reduces this to half the size.